packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -895,54 +752,41 @@ class _TappableLabel {
 
 class _DialPainter extends CustomPainter {
   _DialPainter({
-    @required this.primaryOuterLabels,
-    @required this.primaryInnerLabels,
-    @required this.secondaryOuterLabels,
-    @required this.secondaryInnerLabels,
+    @required this.primaryLabels,
+    @required this.secondaryLabels,
     @required this.backgroundColor,
     @required this.accentColor,
+    @required this.dotColor,
     @required this.theta,
-    @required this.activeRing,
     @required this.textDirection,
     @required this.selectedValue,
   }) : super(repaint: PaintingBinding.instance.systemFonts);
 
-  final List<_TappableLabel> primaryOuterLabels;
-  final List<_TappableLabel> primaryInnerLabels;
-  final List<_TappableLabel> secondaryOuterLabels;
-  final List<_TappableLabel> secondaryInnerLabels;
+  final List<_TappableLabel> primaryLabels;
+  final List<_TappableLabel> secondaryLabels;
   final Color backgroundColor;
   final Color accentColor;
+  final Color dotColor;
   final double theta;
-  final _DialRing activeRing;
   final TextDirection textDirection;
   final int selectedValue;
 
+  static const double _labelPadding = 28.0;
+
   @override
   void paint(Canvas canvas, Size size) {
     final double radius = size.shortestSide / 2.0;
     final Offset center = Offset(size.width / 2.0, size.height / 2.0);
     final Offset centerPoint = center;
     canvas.drawCircle(centerPoint, radius, Paint()..color = backgroundColor);
 
-    const double labelPadding = 24.0;
-    final double outerLabelRadius = radius - labelPadding;
-    final double innerLabelRadius = radius - labelPadding * 2.5;
-    Offset getOffsetForTheta(double theta, _DialRing ring) {
-      double labelRadius;
-      switch (ring) {
-        case _DialRing.outer:
-          labelRadius = outerLabelRadius;
-          break;
-        case _DialRing.inner:
-          labelRadius = innerLabelRadius;
-          break;
-      }
+    final double labelRadius = radius - _labelPadding;
+    Offset getOffsetForTheta(double theta) {
       return center + Offset(labelRadius * math.cos(theta),
-                                 -labelRadius * math.sin(theta));
+          -labelRadius * math.sin(theta));
```

Comment:
this should probably be on the same line as the return, or it could have a trailing comma for better formatting

Reply:
Done